tests/test_cli.py

```diff
@@ -495,7 +495,7 @@ class TestWriteCitations(unittest.TestCase):
     def test_model_citations(self):
         contig = stdpopsim.Contig()
         species = stdpopsim.get_species("homsap")
-        model = species.get_demographic_model("OutOfAfrica_3G09")
+        model = species.get_demographic_model("Constant")
```

Comment:
We need to fix this test so that it's not so strict --- @grahamgower, what can we do here?

Reply:
I think the test is just bogus. It makes no sense to have to change the test code (potentially) every time the model code is changed. I'll send a PR removing the test.